lib/services/nodes-api-service.js

```diff
@@ -280,6 +280,11 @@ function nodeApiServiceFactory(
                     }
                 };
                 return workflowApiService.createAndRunGraph(configuration);
+            } else if (node.type === Constants.NodeTypes.Rack) {
+                return eventsProtocol.publishNodeEvent(node, 'added')
```

Comment:
maybe it's worth to extend to other types of nodes, not only for rack nodes.  if you think it's fine,  https://github.com/RackHD/on-http/blob/master/lib/services/nodes-api-service.js#L284  https://github.com/RackHD/on-http/blob/master/lib/services/profiles-api-service.js#L224  http://rackhd.readthedocs.io/en/latest/rackhd/event_notification.html#node-events  need to  be updated for publish events and update docs.   I also find https://github.com/RackHD/on-http/blob/master/lib/services/profiles-api-service.js#L216  createNodeAndRunDiscovery()  maybe should be located at nodes-api-service.js, not profiles-api-service.js. it could be refined. Sorry for mentioning so much, and out of your original purpose.